### PR TITLE
Fixed tests compilation error and added travis integration.

### PR DESCRIPTION
**Pull request by xp-vit**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,3 +3,10 @@ jdk:
   - oraclejdk7
   - openjdk7
   - openjdk6
+before_install:
```
Comment (bit-man) [inline .travis.yml]: Git and Maven are installed already at Travis. Have you found problems ?

**Comment by bit-man** [comment]
Thanks for your help !!
Could you take a look at my comments about .travis.yml file ?

**Comment by xp-vit** [comment]
Sure! I will make it passing. I actually created this PR in order to check if project still maintained :-) So, I'll update my PR.

File: .travis.yml (hunk 1 above)
Comment (xp-vit) [inline .travis.yml]: Removed git and maven install

**Comment by xp-vit** [comment]
Now Travis build is passing. To achieve this I had to fix some tests and comment 11 of them

**Comment by bit-man** [comment]
Thanks for your contribution, it worked like a charm!

**Pull request merged**